tests/ci/run_tests

```diff
@@ -130,6 +136,12 @@ while getopts "dbfgwstuvxyzijkl" opt; do
         l)
             GANACHE_PROXY_MODEL_WP_SGX_TESTS=1
             ;;
+		a)
+			MULTIPLE_WP_DIRECT_MODEL_TESTS=1
+			;;
+		c)
+			MULTIPLE_WP_DIRECT_MODEL_SGX_TESTS=1
+			;;
```

Comment:
please align the newly added lines to previous lines. Remove extra spaces added at the beginning.